### DISCUSSION TITLE
manager/domain-worker-cdn-and-tunneling/How-to-use-desired-clean-IPs-based-on-a-specific-format/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4509] [Content Help کمک به اصلاح محتوا]
# manager/domain-worker-cdn-and-tunneling/How-to-use-desired-clean-IPs-based-on-a-specific-format/

The proxy manager platform.

https://hiddify.com/manager/domain-worker-cdn-and-tunneling/How-to-use-desired-clean-IPs-based-on-a-specific-format/

**Comment by alexlii1971** [comment]
Hello,

Place the copied template in the Force the use of the following IPs field in the settings related to your domain in the Hidify panel.

I check all of the setting, but did not found the field of "Force the use of the following IPs ".

Could you please let us know where we can find "Force the use of the following IPs"? in which part of setting in he Hiddify Panel please?


Thanks

**Comment by kekomenos** [comment]
> # manager/domain-worker-cdn-and-tunneling/How-to-use-desired-clean-IPs-based-on-a-specific-format/
> The proxy manager platform.
> 
> https://hiddify.com/manager/domain-worker-cdn-and-tunneling/How-to-use-desired-clean-IPs-based-on-a-specific-format/
